srcpkgs/pinephone-utils/patches/vendor/modem-rules.patch

```diff
@@ -0,0 +1,8 @@
+--- device/main/device-pine64-pinephone/90-modem-eg25.rules	2021-01-27 07:11:44.000000000 -0800
++++ srcpkgs/pinephone-utils/files/90-modem-eg25.rules	2020-09-25 01:37:55.390496779 -0700
+@@ -3,5 +3,3 @@
+ SUBSYSTEMS=="usb", KERNEL=="ttyUSB[0-9]*", ATTRS{idVendor}=="2c7c", ATTRS{idProduct}=="0125", ENV{.LOCAL_ifNum}=="01", SYMLINK+="EG25.NMEA", MODE="0660"
+ SUBSYSTEMS=="usb", KERNEL=="ttyUSB[0-9]*", ATTRS{idVendor}=="2c7c", ATTRS{idProduct}=="0125", ENV{.LOCAL_ifNum}=="02", SYMLINK+="EG25.AT", MODE="0660"
+ SUBSYSTEMS=="usb", KERNEL=="ttyUSB[0-9]*", ATTRS{idVendor}=="2c7c", ATTRS{idProduct}=="0125", ENV{.LOCAL_ifNum}=="03", SYMLINK+="EG25.MODEM", MODE="0660"
+-
+-ACTION=="add", SUBSYSTEM=="tty", ATTRS{idVendor}=="2c7c", ATTRS{idProduct}=="0125", ENV{.LOCAL_ifNum}=="02", RUN+="/usr/bin/pinephone_setup-modem '%E{DEVNAME}'"
```

Comment:
Can you explain the reason for the patch at the top?

Reply:
@ericonr Initially I took this out because I noticed this invocation didn't do anything, but that might be because the upstream script is 'broken' in a sense. The main parts of it are tested, but the variable which contains the path to the modem device file is unset https://gitlab.com/postmarketOS/pmaports/-/blob/master/device/main/device-pine64-pinephone/setup-modem.sh

This might have gone unnoticed, since the changes made by this script should be persistent until new settings are uploaded to the modem. I'll ask the pmos people about it, and try the script again myself. If nothing is wrong, I'll just kill this patch.